src/main/java/by/custom/utilcalculator/domain/Price.java

```diff
@@ -0,0 +1,26 @@
+package by.custom.utilcalculator.domain;
+
+import by.custom.utilcalculator.domain.constants.CurrencyType;
+
+public class Price {
+    private final double amount;
```

Comment:
Think about using `BigDecimal` instead double. It's not a fact that it's really needed.
https://medium.com/@cancerian0684/which-data-type-would-you-choose-for-storing-currency-values-like-trading-price-dd7489e7a439